src/ucs/vfs/base/vfs_obj.c

```diff
@@ -80,11 +81,17 @@ struct {
     }
 
 
-void ucs_vfs_memory_address_show_cb(void *obj, ucs_string_buffer_t *strb)
+void ucs_vfs_memory_address_show_cb(void *obj, void *arg,
+                                    ucs_string_buffer_t *strb)
 {
     ucs_string_buffer_appendf(strb, "%p\n", obj);
 }
 
+void ucs_vfs_show_string(void *obj, void *arg, ucs_string_buffer_t *strb)
```

Comment:
can we make naming consistent, so that `ucs_vfs_show_string` and `ucs_vfs_memory_address_show_cb` would use the same naming rules?

Reply:
done